OJP/OJP_Requests.xsd

```diff
@@ -223,7 +295,13 @@
 	<xs:annotation>
 		<xs:documentation>========================================== TripRefine Service ==========================================</xs:documentation>
 	</xs:annotation>
-	<xs:element name="OJPTripRefineRequest" type="OJPTripRefineRequestStructure" substitutionGroup="siri:AbstractFunctionalServiceRequest"/>
+	<xs:element name="OJPTripRefineRequest" type="OJPTripRefineRequestStructure" substitutionGroup="siri:AbstractFunctionalServiceRequest">
+		<xs:annotation>
+			<xs:documentation>The request does not depend on the assumption that the trip has been retrieved from the same server; it may, in fact, even stem from another source than a OJP system. 
```

Comment:
```suggestion
			<xs:documentation>The trip refinement service retrieves additional or updated information (e.g., real-time data) about a given, previously retrieved trip. It does not depend on the assumption that the trip has been retrieved from the same server; it may, in fact, even stem from another source than a OJP system. 
```